lqdoj_backend/settings.py

```diff
@@ -55,19 +56,14 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.TokenAuthentication',
+    )
+}
+
 CORS_ORIGIN_ALLOW_ALL = True
-CORS_ALLOW_CREDENTIALS = True
-CORS_ALLOW_HEADERS = [
-    'accept',
-    'accept-encoding',
-    'authorization',
-    'content-type',
-    'dnt',
-    'origin',
-    'user-agent',
-    'x-csrftoken',
-    'x-requested-with'
-]
+CORS_ALLOW_CREDENTIALS = False
```

Comment:
The default value is False so we don't have to explicitly set it to False (reference: https://github.com/adamchainz/django-cors-headers#cors_allow_credentials)